lib/users.js

```diff
@@ -92,18 +92,20 @@ users.setDBPassword = async (userInfo, newPW) => {
   }
   const {Accounts} = db.getModels()
   try {
-    Accounts.update({
+    const updatedAccount = await Accounts.update({
       dbPassword: newPW
     }, {
       where: {
         username: userInfo.username
-      }
+      },
+      returning: true
     })
+    logger.info('DB password has set successfully', userInfo.username)
+    return updatedAccount[1][0]
```

Comment:
What does the updated account look like? Why are you only accessing index 1?

Reply:
I figured it out from the test but could you add a comment for future engineers?

Reply:
Okay I'll do tonight